js/impact.js

```diff
@@ -1381,8 +1381,8 @@ var GLPIImpact = {
             },
             {
                 id             : 'editEdge',
-                content        : '<i class="fas fa-edit me-2"></i>' + _.escape(__("Edge properties...")),
-                tooltipText    : _.escape(__("Set name for this edge")),
+                content        : '<i class="fas fa-edit me-2"></i>' + __("Edge properties..."),
+                tooltipText    : _.unescape(__("Set name for this edge")),
```

Comment:
Why `unescape` here instead of just displaying the result of `__()` like the line above ?

Reply:
Because cytoscape will escape it.

Reply:
`content` expects an HTML string and `tooltipText` expects a non HTML string.